unit_tests/test_nemesis.py

```diff
@@ -151,3 +153,21 @@ def test_list_topology_changes_monkey():
     for disrupt_method in collected_disrupt_methods_names:
         assert disrupt_method in expected_disrupt_method_names, \
             f"{disrupt_method=} from {collected_disrupt_methods_names=} was not found in {expected_disrupt_method_names=}"
+
+
+def test_disabled_monkey():
+
+    ToggleGcModeMonkey.disabled = True
+
+    tester = FakeTester()
+
+    all_disrupt_methods = {attr[1].__name__ for attr in inspect.getmembers(Nemesis) if
+                           attr[0].startswith('disrupt_') and
+                           callable(attr[1])}
+
+    tester.params["nemesis_include_filter"] = ['!disabled']
+    sisphus = FakeSisyphusMonkey(tester, None)
+
+    collected_disrupt_methods_names = {disrupt.__name__ for disrupt in sisphus.disruptions_list}
+    assert 'disrupt_toggle_table_gc_mode' not in collected_disrupt_methods_names
```

Comment:
this validation might be redundant and included in the next line validation, no?

Reply:
yes, it too me a while to get the 2nd one working, cause there was a bug, and on nemesis wasn't collected.

yes this can be removed

Reply:
removed